sklearn/utils/graph.py

```diff
@@ -12,6 +12,7 @@
 
 import numpy as np
 from scipy import sparse
+from scipy.sparse.csgraph import laplacian as graph_laplacian
```

Comment:
pyflake raises F401: imported but unused on this line.

However, it *is* used from other files, just not from this one.

I am inclined to add #noqa and keep providing it this way.

Alternative 2: fix **our** codebase to directly import from scipy, and provide a thin function here that raises deprecation warnings (in case users are calling this directly.)

after some thought, alternative 2 sounds better. But it was healthy to have a run of CI with this simple commit.